java/image.bzl

```diff
@@ -155,30 +155,13 @@ def _jar_dep_layer_impl(ctx):
     # type option here. The app layer would also need to be updated to
     # consider data flies include in the dep_layer as "available" so they
     # aren't duplicated in the top layer.
-    return app_layer_impl(ctx, runfiles = java_files_with_data)
+    return lang_image.implementation(ctx, runfiles = java_files_with_data)
 
 jar_dep_layer = rule(
-    attrs = dicts.add(_container.image.attrs, {
-        # The base image on which to overlay the dependency layers.
-        "base": attr.label(mandatory = True),
-
-        # The binary target for which we are synthesizing an image.
-        "binary": attr.label(mandatory = False),
-        # Set this to true to create an empty workspace directory under the
-        # app directory specified as the 'directory' attribute.
-        "create_empty_workspace_dir": attr.bool(default = False),
-        # https://github.com/bazelbuild/bazel/issues/2176
```

Comment:
This comment and link to the issue is still valid. Let's move it to the comment in the lang/image.bzl.

Reply:
Done!